src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -210,6 +210,7 @@
                     outOfDate: false} : null
             },
             async importDataset() {
+                if(this.newDatasetId){ 
```

Comment:
This needs auto-formatting/proper indenting! And a unit test. Can we also mark the button as disabled when `this.newDatasetId` is null, so the user can't even click it (and add a unit test for this too)?

Reply:
I auto-format, I think the issue has to do with how IntelliJ and VSCode do formatting. I will explore IntelliJ for formatting.